.github/pull_request_template.md

```diff
@@ -36,6 +29,14 @@ https://kamma.atlassian.net/browse/GET-XXXX
 ## Checklist
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+
+### Code Style
 - [ ] My code follows the code style of this project.
 - [ ] My change requires a change to the documentation.
 - [ ] I have updated the documentation accordingly.
+
+### Security
+
```

Comment:
Missing the DB line, think its been accidentally deleted between commits

```suggestion

- [ ] All raw database queries use prepared statements
```

Reply:
I didn't realise. Thanks @othyn .